doc/source/setup-helm.rst

```diff
@@ -59,6 +64,22 @@ shell from your provider), enter:
 
 This command only needs to run once per Kubernetes cluster.
 
+.. note::
+
+   The local and remote version of ``helm`` must be the same in order to
+   ensure they can talk to each other. If you wish to run ``helm`` commands
+   from a *new* computer than the one used to run the commands above, you
+   must re-initialize it by running the following modified version of
+   the ``init`` command:
+
+   ``helm init --client-only --service-account tiller``
+
+   This will initialize ``helm`` locally, according to the version that is
+   running remotely on the cluster. Note that this requires ``kubectl``
```

Comment:
With `--client-only` I'm pretty sure helm won't connect to the cluster, and you have to download the correct version.

Reply:
ah no, you're probably right, I was just optimistically assuming :-)

Reply:
that said, any reason *not* to just run `helm init` again? I guess if your local helm has a different version maybe it'll update the cluster version and invalidate other people's helm connection?

Reply:
Yes, that's a good reason not to. If someone is new to z2jh they're probably testing on their own from a single computer so this isn't relevant, and if multiple people/computers are involved it's probably going beyond just a test installation so I think it's better to be safe.

Reply:
I would leave it as written. Perhaps remind again `helm version` will output the version of the client and the server.